dist/scrollnav.min.js

```diff
@@ -1,2 +1,2 @@
.classList.remove("scroll-nav__item--active"),y.removeAttribute("data-sn-active")),w.classList.add("scroll-nav__item--active"),w.setAttribute("data-sn-active",!0))}else y&&(y.classList.remove("scroll-nav__item--active"),y.removeAttribute("data-sn-active"))}(w,f.nav),w}return window.addEventListener("scroll",h),h}function B(f){return f instanceof Element}module.exports={init:function(h,_){if(this.settings=f({sections:"h2",insertTarget:h,insertLocation:"before",easingStyle:"easeOutQuad",updateHistory:!0},_),B(h))if(!this.settings.insertTarget||B(this.settings.insertTarget))if(["append","prepend","after","before"].includes(this.settings.insertLocation)){var I,Q,C,F,R=h.querySelectorAll(this.settings.sections);if(R.length)return this.data=y(R,this.settings),this.nav=w(this.data),Q=(I=this).settings.insertTarget,"append"===(C=I.settings.insertLocation)?Q.appendChild(I.nav):"prepend"===C?Q.insertBefore(I.nav,Q.firstChild):"before"===C?Q.parentNode.insertBefore(I.nav,Q):"after"===C&&Q.parentNode.insertBefore(I.nav,Q.nextSibling),O=function(f){var h=f.settings;function y(y){y.preventDefault();var w=.39*window.innerHeight;return M(L(y.target,f.data)-w,h.easingStyle).then(function(){h.updateHistory&&history.replaceState({},"",y.target.getAttribute("href")),h.onScroll&&h.onScroll()})}return f.nav.querySelectorAll("a").forEach(function(f){f.addEventListener("click",y)}),y}(this),x=q(this),j=function(f){function h(){f.data=E(f.data)}return window.addEventListener("resize",h),h}(this),this.settings.debug&&((F=document.createElement("div")).className="snDebugger",F.setAttribute("style","\n      position: fixed;\n      top: 40%;\n      height: 0px;\n      border-bottom:5px solid red;\n      border-top: 5px solid blue;\n      width: 100%;\n      opacity: .5;\n      pointer-events: none;\n    "),document.body.appendChild(F)),this.settings.onInit?this.settings.onInit():void 0;this.settings.debug&&console.error('\n        scrollnav build failed, could not find any "'+this.settings.sections+'"\n        elements inside of "'+h+'"\n      ')}else this.settings.debug&&console.error('\n        scrollnav build failed, options.insertLocation "'+this.settings.insertLocation+'" is not a valid option\n      ');else this.settings.debug&&console.error('\n        scrollnav build failed, options.insertTarget "'+h+'" is not an HTML Element\n      ');else this.settings.debug&&console.error('\n        scrollnav build failed, content argument "'+h+'" is not an HTML Element\n      ')},destroy:function(h){if(this.settings=f(this.settings,h),function(f,h){f.querySelectorAll("a").forEach(function(f){f.removeEventListener("click",h)})}(this.nav,O),function(f){window.removeEventListener("scroll",f)}(x),function(f){window.removeEventListener("resize",f)}(j),this.nav.remove(),this.settings.onDestroy)return this.settings.onDestroy()},updatePositions:function(h){if(this.settings=f(this.settings,h),this.data=E(this.data),this.settings.onUpdatePositions)return this.settings.onUpdatePositions()}};
```

Comment:
No need to include the minified code, that'll be bundled up on release. Can you revert these please?